backend/dal/Helpers/Extensions/ParcelExtensions.cs

```diff
@@ -37,5 +41,104 @@ public static void ThrowIfPidPinUpdated(this DbSet<Entity.Parcel> parcels, Entit
 
             if (pidPinUpdated) throw new DbUpdateException("PID and PIN may not be updated.");
         }
+
+        /// <summary>
+        /// Generate a query for the specified 'filter'.
+        /// </summary>
+        /// <param name="context"></param>
+        /// <param name="user"></param>
+        /// <param name="filter"></param>
+        /// <returns></returns>
+        public static IQueryable<Entity.Parcel> GenerateQuery(this PimsContext context, ClaimsPrincipal user, Entity.Models.ParcelFilter filter)
```

Comment:
What do you think about trying to move/group some of these query components to common functions? for example lat/lon and evaluations are probably going to work the same between multiple different filters. Not required for this review, just thinking.

Reply:
I did already start that process by pulling them into extension methods.  But yes I like going one step further (when I find time).